src/djangosaml2_spid/conf.py

```diff
@@ -0,0 +1,278 @@
+import os
```

Comment:
Ho appena testato che pur modificando `spid_config.spid_settings` continua a caricare quanto definito qui, su questo dobbiamo tornarci sopra con dei test. Farò dei commit al di sopra dei tuoi prima di unire su dev, poi ripartiremo da li

Reply:
I test eseguiti con _runtests.py_ non usano i settings di `spid_config.spid_settings`, che sono usati solo nella demo, ma usano la configurazione `tests/test_settings.py`.

Se si eseguono i test con `example/manage.py` si caricano invece i settings di  `spid_config.spid_settings`. 

Per quanto riguarda questa configurazione dovrebbero essere già usate tutte le configurazioni _SPID_*_ ad esclusione di SPID_BASE_SCHEMA_HOST_PORT e SPID_BASE_URL, che sono state tolte perché il valore effettivo, se deve corrispondere, dipende dall'ambiente di esecuzione. Se non deve corrispondere all'effettivo valore di host e port si può anche ripristionarle magari con una URL completamente slegata dal run come ad esempio 'http://spid.test'.

La configurazione SAML_CONFIG inserita in _spid_settings.py_ viene considerata solo nelle voci non determinate dalle configurazioni _SPID_*_ ('organization', 'debug', 'xmlenc_binary'). Si potrebbe anche optare per mantenere il più possibile, aggiornando solo quello che è vincolato dalle configurazioni SPID, ma sarebbe comunque  necessario impostare altri valori (es ['service']['sp'][''required_attributes']) necessari al funzionamento di SPID.

L'obiettivo sarebbe quello di avere una configurazione di default costruita secondo le modalità previste da djangosaml2, lasciando poi all'utente l'onere solo di inserire le voci _SPID_*_ obbligatorie e volendo quelle opzionali. Una configurazione SAML_CONFIG fissa sarebbe peraltro difficile da gestire in progetti che dovessero avvalersi di altri sistemi di autenticazione federata.

Reply:
Ho integrato qualche generalizzazione aggiuntiva senza rompere l'approccio da te proposto, che ormai è adottato in pieno, fai un test su Dev branch. Ad oggi sono ben due le release ufficiali che adottano il tuo contributo. Grazie infinite